wordpress-mobile/WordPress-Android
# Jetpack Story block bundling - internal test release

This PR made for preparing an internal APK to test so:
- we lift the __DEV__ flag on mobile-gutenberg
- make Story the only jetpack block available

To test:
- run `./gradlew assembleZalphaDebug`
- verify the resulting APK shows the Story beta block in the picker and is able to render Story blocks in Gutenberg.

Related PRs
Gutenberg: https://github.com/WordPress/gutenberg/pull/26247
Gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2738


PR submission checklist:
